### ISSUE TITLE
Would it be possible to support llama index?

### ISSUE THREAD

**Issue by sf17** [cocktailpeanut/dalai#304]
Would it be possible to support llama index?
Or anything else based on https://github.com/hwchase17/langchain ?

See https://github.com/jerryjliu/llama_index/issues/928
This would allow for easier extensibility.

**Comment by claysauruswrecks** [comment]
Yes, but probably not in this project. Your best bet is to write your application to interact with the API.

**Comment by ansarizafar** [comment]
I would also love to use llama index with dalai. @cocktailpeanut whats your take on this issue?
